packages/trader/src/Modules/CFD/Containers/props.types.ts

```diff
@@ -1,8 +1,11 @@
+import { FormikHelpers as FormikActions } from 'formik';
+import { TCFDPasswordFormValues } from './cfd-password-modal';
+
 export type TCFDChangePasswordConfirmationProps = {
-    confirm_label: string;
+    confirm_label?: string;
```

Comment:
ChangePasswordConfirmation is used without this prop inside CFDCreatePasswordForm, thus it's not a required prop 🙂